api/app/Models/Permission.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Spira\Model\Model\BaseModel;
+
+class Permission extends BaseModel
```

Comment:
This should probably extend `VirtualModel` as it has already disabled the methods/properties that make it an eloquent table-bound model.